tests/test_mutation.py

```diff
@@ -406,8 +412,15 @@ async def test_returning_all_columns_and_relations(
         session: Client,
         session_async: AsyncClient,
     ):
+        user_uuid = uuid4()
         expected_authors = [
-            author.copy(update={"age": author.age + 1})
+            author.copy(
+                update={
+                    "age": author.age + 1,
+                    "updated_by": user_uuid,
```

Comment:
Confused why this got added?

Reply:
sharp eyes! Before, tests that used the `all_columns()` helper function did not compare any `updated_at`, `updated_by`, etc (see this [change](https://github.com/hotaru355/cuckoo-hasura/pull/6/files#diff-02c3a3a504afaef7a801cd71c4511b14ea2ccdeb0d0c876249bf4335b2a8306dR370) ). I changed that helper to really get _all_ columns to validate the new feature and that broke some test like this one.